content/developer/tutorials/git.rst

```diff
@@ -0,0 +1,61 @@
+=====================
+Collaborate using git
+=====================
+
+Introduction
+============
+
+*What are git and github, what are there for*
+
+Setup git and github
+====================
+
+*Basically a link to the "Environment setup" section of the contributing/develoment*
+
+There are a few things on that page that I don't like
+
+* the windows installation wizard is super long and complicated (14 setup screens), especially to newcomers, there is no guide in the git book nor on the official https://gitforwindows.org/ website to help newcomers install git, which are the two documents referenced by our tutorial at the moment
```

Comment:
If there is a better URL to share, it could totally be done.

Reply:
I haven't found one yet... for my classes at uni I ended up writting my own tutorial